runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/ShuffleSink.java

```diff
@@ -238,7 +238,7 @@ public long add(WindowedValue<T> windowedElem) throws IOException {
           // key by concatenating the nested encoding of sortKey and outer encoding of the
           // timestamp. An alternative implementation would be to use OrderedCode but it
           // is unnecessary here because the nested encoding of sortKey achieves the same effect,
-          // due to the nested encoding being a https://en.wikipedia.org/wiki/Prefix_code.
+          // due to the nested encoding being an https://en.wikipedia.org/wiki/Prefix_code.
```

Comment:
Not sure about this one! :D

Reply:
an https

Reply:
I had to read it a few times because my brain automatically "pronounces" `an https://en.wikipedia.org/wiki/Prefix_code`  as `an Prefix Code` -- I would have no objection either leaving it or changing it!

Reply:
O(∩_∩)O~

Reply:
I would leave it as "a" in this case.